measures/ResStockArguments/resources/constants.rb

```diff
@@ -70,7 +70,7 @@ module Constants
                                    'electric_panel_load_water_heater_power',
                                    'electric_panel_load_clothes_dryer_power',
                                    'electric_panel_load_dishwasher_power',
-                                   'electric_panel_load_cooking_range_power',
+                                   # 'electric_panel_load_cooking_range_power',
```

Comment:
This is commented out because cooking range power comes from options_lookup right?

Reply:
Correct. Fuel options assign a zero, electric resistance a 12K, and electric induction a 10K.

Reply:
Looking back at our OS-HPXML defaults for Power Rating, I don't remember why we didn't just include an additional, e.g., `rangeoven_induction`, row for electric induction cooking range?

![image](https://github.com/user-attachments/assets/02067bda-c245-46e5-ab26-25d4ae5eb71d)

Reply:
Note I decided to take this approach. See https://github.com/NREL/OpenStudio-HPXML/pull/1946/commits/95ffa598721e21fc073972306f2c73592e13c529.

Reply:
We used to assume 12 kW for both ER and induction cooking, hence there was just one OS-HPXML default value.

The rationale for getting power rating from options_lookup instead of default is that we can more easily introduce diversity and options from options_lookup, especially given that cooking load is quite large and cooking equipment do vary quite a bit in size and power requirement.

I think for this round, we can rely on default values. In the future, we may want to allow appliance power ratings to be passed in from option_lookup to override the defaults.

Reply:
Yes, once we want to actually override the defaults, we can use the lookup to do that -- it _is_ currently allowed. We just currently don't have any options that assign values different than the default values.

Reply:
Used in https://github.com/NREL/OpenStudio-HPXML/pull/1946/commits/1b7843050a19595179ae27f1cb6134f48650c06d.

The approach is basically to (1) use the EV power override argument if specified, otherwise (2) use the EVc charging power value if specified, otherwise (3) use OS-HPXML defaults.